docs/getting-started.mdx

```diff
@@ -408,6 +437,68 @@ up. To run `integration_test/example_test.dart` on a connected Android, iOS or m
           internals depend on it.
       </Error>
 
+### FAQ
+
+<Accordion title="Testing fails with error inside 'integration_test/test_bundle.dart'">
+  The reason is probably a mismatch of `patrol` and `patrol_cli` versions. Make sure that you 
+  use the latest versions of both packages.
+</Accordion>
+
+<Accordion title="'example_test.dart' passed but I can't get my application to run 
+within the patrol test.">
+  To run your application within the patrol test, you need to call `$.pumpWidgetAndSettle()`,
+  and pass your application's main widget to it. Be sure that you registered all the 
+  necessary services before calling `$.pumpWidgetAndSettle()`.
+  Here's the example of running an app within the patrol test:
+  ```dart
+  void main() {
+    patrolTest('real app test', ($) async {
+    
+    // Do all the necessary setup here (DI, services, etc.)
+
+    await $.pumpWidgetAndSettle(const MyApp()); // Your's app main widget
+
+    // Start testing your app here
+
+    });
+  }
+  ```
+  It's a good practice to create a setup wrapper function for your tests, so you don't have to
+  repeat the same code in every test.
+</Accordion>
+
+
+#### Android
+
+<Accordion title="Where do I get 'package_name' from?">
+  Go to `android/app/build.gradle` and look for `applicationId` in `defaultConfig` section.
+</Accordion>
+
+<Accordion title="Running patrol test fails containing 'Unsupported class file major version' error">
+  It's most likely caused by using incompatible JDK version. 
+  Change your $JAVA_HOME path to use JetBrainsRuntime.
+</Accordion>
```

Comment:
This isn't explained sufficiently. People will still come asking "what that means, how do I do that?".

You can tell users to run `javac -version` to learn what version of Java compiler they're using:

```
$ javac -version
javac 17.0.9
```

Say that we require 17 (just as newest AGP):

![Screenshot 2023-12-11 at 10 48 11 AM](https://github.com/leancodepl/patrol/assets/40357511/bbd056ad-4505-447a-85d8-e84a901c5286)

Also it's not required to use JetBrains runtime, and many users won't have it (if they don't have AS/IntelliJ installed). Any OpenJDK-conforming JDK will work. But you can give an example where one can find JetBrains Runtime if they have AS/IntelliJ installed.

Also [link to this page](https://developer.android.com/build/jdks) ("Learn more").